dss/operations/lambda_params.py

```diff
@@ -0,0 +1,291 @@
+"""
+Get and set environment variables in deployed lambda functions using the SSM param store
+variable named "environment".
+"""
+import os
+import sys
+import select
+import json
+import argparse
+import logging
+import typing
+
+from botocore.exceptions import ClientError
+
+from dss.operations import dispatch
+from dss.operations.ssm_params import get_ssm_variable_prefix, fix_ssm_variable_prefix
+from dss.operations.ssm_params import get_ssm_parameter, set_ssm_parameter
+
+from dss.util.aws.clients import ssm as ssm_client  # type: ignore
+from dss.util.aws.clients import secretsmanager as sm_client  # type: ignore
+from dss.util.aws.clients import es as es_client  # type: ignore
+import dss.util.aws.clients
+lambda_client = getattr(dss.util.aws.clients, "lambda")
+
+
+logger = logging.getLogger(__name__)
```

Comment:
We might consider using `print` instead of the logger throughout this file since this script will always be executed on an operators machine.

Note that this is not the case for some of the other operational scripting, which may be executed in lambda functions and therefore needs to output log messages.

Reply:
Using `print()` everywhere now, instead of mixing `print` and `logger`